user/common-build-problems.md

```diff
@@ -610,3 +610,36 @@ jobs:
 
 
 This creates only one job,  _Peanut Butter and Bread_ under the stage named _Breakfast_ as you have defined. It is important to note that in YAML, the `-` symbol is used to create a list of items and the earlier example creates a list of 2 items, while you actually wanted 1. You can read more on [How to define Build Stages](/user/build-stages/#how-to-define-build-stages) and YAML lists syntax in the official [documentation](https://yaml.org/spec/1.2/spec.html#id2759963).
+
+## Build terminates early if I use `exit` in my script
+
+Check the `scrip` section in your .travis.yaml configuration file. You might have used `exit` command in the given script
+
+i:e
```

Comment:
You probably meant "for example".

---
### Aside
The phrase "_i.e._" (not "_i:e_") is shorthand for _id est_, meaning "in other words" to introduce another equivalent idea afterwards, and it does not fit here.

If you do mean "for example", the abbreviation is "_e.g._", but it should not start the sentence. Just write out the words.